apisix/plugins/cors.lua

```diff
@@ -178,6 +179,9 @@ function _M.check_schema(conf, schema_type)
             end
         end
     end
+    if conf.allow_origins == "" and (not conf.allow_origins_by_regex or next(conf.allow_origins_by_regex) == nil) then
```

Comment:
As allow_origins_by_regex is an array, we can use `#` to check if it's empty

Reply:
Thanks @spacewander I knew there should be a better way but my lua foo is not very good :slightly_smiling_face: 

I will update this once we are clear on the `metadata_schema` item.